services/clusters-keeper/tests/unit/conftest.py

```diff
@@ -397,3 +400,17 @@ def clusters_keeper_docker_compose() -> dict[str, Any]:
     )
     assert data
     return yaml.safe_load(data)
+
+
+@pytest.fixture(scope="session")
+def clusters_keeper_namespace() -> RPCNamespace:
+    return parse_obj_as(RPCNamespace, "clusters-keeper")
```

Comment:
THOUGHT: I added these in `model_library` to reduce the errors ... perhaps the names of the exposed functions should also be there (as is it would be some sort of openapi). For instance
https://github.com/ITISFoundation/osparc-simcore/blob/master/packages/models-library/src/models_library/api_schemas_payments/__init__.py#L7

Reply:
ok that makes sense.